docs/src/content/callbacks.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: 'Callbacks'
+---
+
+import Examples from 'components/examples'
+import Contents from 'components/table-of-contents'
+import { SimplePanel } from 'components/panel'
+import { Parameters, Parameter } from 'components/parameters'
+import { H1, H2, H3, H4, H5 } from 'components/core/heading'
+
+# Accepting callbacks
+
+A callback lets you receive messages from Notify to a URL you choose.
+
+<span />
+
+On a server you control, you should set up an endpoint that can accept POST
+requests. Notify will POST messages in a JSON format to the endpoint you specify.
+
+To ensure that callbacks are coming from Notify and not some other malicious
+source, you will need to create a "bearer token" when setting up the callback
+URL. A bearer token should be a long randomised piece of text that an attacker
+wouldn't be able to impersonate.
+
+**When creating a bearer token, you should:**
+
+- keep your bearer token secure,
+- change it if you have any reason to think it might no longer be trusted,
+- make sure that callbacks you recieve from Notify contain your bearer token as an authorisation header,
+- and use a hashed value so that Notify doesn't hold the true token.
+
+<Contents />
+
+## Message delivery updates
+
+When you send an email or text message through Notify, we can send a receipt to your callback URL to tell you if we were able to deliver it or not.
+
+```json inline nonumbers
```

Comment:
@dagoof should you add an intro above this like `In JSON, the messages you receive will look something like this:`?